blackboxopt/optimizers/testing.py

```diff
@@ -124,7 +122,8 @@ def is_deterministic_with_fixed_seed(optimizer_class, optimizer_kwargs: dict) ->
         )
 
         es1 = opt.get_evaluation_specification()
-        opt.report_evaluation(es1.create_evaluation(objectives={"loss": 0.42}))
+        evaluation1 = es1.create_evaluation(objectives={"loss": 0.42})
+        opt.report([evaluation1])
```

Comment:
Is this intended to cover testing the batch report functionality?
I'm wondering whether it might make sense to add a separate test for that. Because if the reproducibility test fails due to an optimizer not supporting report many evaluations, this is unnecessarily hard for the developer to find out, when instead there could be a `handles_reporting_evaluations_list` which immediately from its name suggests what might have gone wrong when it fails.

Reply:
Actually, no, it's not intended to cover batch functionality, I just forgot to remove the list. ;-)
But adding a test for batch might be a good idea, so I'll do it :-)